<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_mainWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>731</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_mainMenu">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button_planATrip0">
         <property name="text">
          <string>plan a trip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_searchForStadiums0">
         <property name="text">
          <string>search for stadium</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_viewStadiums">
         <property name="text">
          <string>View stadiums</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_shop0">
         <property name="text">
          <string>shop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_adminAccess0">
         <property name="text">
          <string>Admin</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_mainMenu0">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>251</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QTextBrowser" name="textBrowser_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>211</width>
          <height>291</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Temp - still thinking about what is going to go here&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8"/>
     </widget>
     <widget class="QLabel" name="label_titleMainMenu">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>MLB Vacation Planer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_planATrip0">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>451</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="button_VisitAll0">
         <property name="text">
          <string>visit all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_customTrip0">
         <property name="text">
          <string>custom trip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_searchForStadiums1">
         <property name="text">
          <string>search for a stadium</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_planATrip0">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>211</width>
        <height>371</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_9">
       <widget class="QTextBrowser" name="textBrowser_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>141</width>
          <height>271</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Temp - still thinking about what is going to go here&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_takeATrip0"/>
     </widget>
     <widget class="QPushButton" name="button_back0">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_mainMenu0">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_planATripTitle">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Plan A Trip</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>371</y>
        <width>158</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_planATrip1">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>361</width>
        <height>301</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Picture  of graph goes here&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;could be a stacked widget&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_outputMilageLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Mileage traveled (in miles): </string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>121</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="button_shop1">
         <property name="text">
          <string>shop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_searchForStadiums2">
         <property name="text">
          <string>search for stadium</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_help2">
         <property name="text">
          <string>help (doesnt work yet)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_takeATrip1">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>201</width>
        <height>381</height>
       </rect>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>141</width>
          <height>301</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Temp - Still thinking about what to do here&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
     </widget>
     <widget class="QPushButton" name="button_back1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_mainMenu1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_searchForStadium0">
     <widget class="QStackedWidget" name="stackedWidget_searchForStadium0">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>90</y>
        <width>191</width>
        <height>271</height>
       </rect>
      </property>
      <widget class="QWidget" name="page_searchForStadium">
       <widget class="QTextBrowser" name="textBrowser_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>141</width>
          <height>151</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Temp - Still thinking about what to do here&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6"/>
     </widget>
     <widget class="QListWidget" name="listWidget_searchResults0">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>491</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_back2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_mainMenu2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_teamSearchTitle">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Team Search</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_searchLine0"/>
       </item>
       <item>
        <widget class="QPushButton" name="button_search0">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_shop0">
     <widget class="QListWidget" name="listWidget_searchResults1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>611</width>
        <height>291</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>On selecting an element, it will launch the team info page which will have a shop functionality</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_shopTitle">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Shop</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_searchLine2"/>
       </item>
       <item>
        <widget class="QPushButton" name="button_search2">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="button_mainMenu3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_back3">
     <widget class="QWidget" name="layoutWidget">
     <widget class="QPushButton" name="button_back3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_stadiumList">
     <widget class="QListWidget" name="listWidget_stadiumList0">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>651</width>
        <height>281</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>This list will contain selectable stadiums</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_stadiumListTitle">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>List of All Stadiums</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_back4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_mainMenu4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_adminLogin0">
     <widget class="QPushButton" name="button_login">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_adminLoginTitle">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Admin Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_mainMenu5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>251</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLineEdit" name="lineEdit_username"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_password"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 </widget>
</widget>
</ui>
